# 재무기반 분석의 소개

```{tableofcontents}
```

재무기반 주식 분석은 기업의 상황과 실적을 나타내는 재무제표 데이터를 기반으로 기업의 가치를 평가하는 분석론이다. 많이 사용되는 것으로 다음과 같은 3가지 방법론이 있다.

* 주가배수 비교 방법론 (price-multiple method)
* 자산기반 가치평가 방법론 (asset-based valuation)
* 펀더멘탈 분석 방법론 (fundamental analysis)

각 방법의 난이도와 신뢰도는 다음과 같다.

| 방법론 | 난이도 | 신뢰도 |
| :-: | :-: | :-: |
| 주가배수 비교 방법론 | 매우 쉬움 | 낮음 |
| 자산기반 가치평가 방법론 | 매우 어려움 | 편차가 심함 |
| 펀더멘탈 분석 방법론 | 어려움 | 높음 |

여기에서는 각 방법의 기본적인 내용을 설명한다. 이후에 자세히 서술하겠지만 주가배수 비교 방법론은 신뢰도가 높지 않고 자산기반 가치평가 방법론는 실제로 수행하기가 어려워 널리 사용되지 않는다. 따라서 두 가지 방법론에 대해서는 이 장에서 간단한 내용만을 설명하고 이후에는 가장 보편적으로 사용되는 재무기반 주식 분석 방법인 펀더멘탈 분석에 대해 자세히 다루도록 한다.

## 주가배수 비교 방법론

### 주가배수

<b>주가배수</b>(price-multiple 또는 단순히 multiple)는 주가를 재무상 중요한 숫자로 나눈 값을 말한다. 보통 다음과 같은 3가지 주가배수가 많이 사용된다.

* PSR(Price Sales Ratio) : 주가를 주당 매출액(sales per share)으로 나눈 값
* PER(Price Earning Ratio) : 주가를 주당 순이익(EPS: earning per share)으로 나눈 값
* PBR(Price Book-value Ratio) : 주가를 주당 장부가(BPS: book-value per share)로 나눈 값

주가배수를 구할 때 주가를 주당 재무수치로 나눌 수도 있고 주가 대신 주가와 주식의 수를 곱한 시가총액(market capitalization)을 사용할 수도 있다. 시가총액을 사용하면 주당 재무수치가 아니라 재무제표에 나온 재무수치를 바로 사용할 수 있다. 예를 들어 PSR, PER, PBR은 다음과 같이 구한다.

* PSR = 시가총액 / 전체 매출액
* PER = 시가총액 / 전체 순이익
* PBR = 시가총액 / 전체 장부가치

이 외에도 현금흐름이나 EBIT(Earning Before Interest and Tas), EBITDA(Earning Before Interest, Tax, Depreciation, and Amortization) 등의 다양한 재무 수치를 사용한 주가배수가 있을 수 있다. 또한 좀 더 신뢰성 있는 분석을 위해 미래의 재무수치를 예측하여 사용하기도 한다. 예를 들어 PER도 다음과 같이 여러가지가 있을 수 있다.

- 후행(trailing) PER = 시가총액 / 최근 결산이익
- 진행(rolling) PER = 시가총액 / 최근 4분기 이익의 평균
- 선행(forward/leading) PER = 시가총액 / 차기 결산이익의 추정치

### 주가배수 비교 방법론의 개요

주가배수 비교 방법론의 기본 가정은 비슷한 기업들은 비슷한 주가배수를 가진다는 것이다. 따라서 주가배수 비교 방법을 사용하려면 우선 분석하고자 하는 대상기업과 유사한 기업 즉, 비교기업군을 찾아야 한다. 일단 비교기업군이 갖추어지면 다음과 같은 두가지 주가배수 비교 분석을 할 수 있다.

* 주가배수기반 주식가치평가 (multple-based valuation)
* 주가배수기반 주식선별 (multiple screening)

### 주가배수기반 주식가치평가 

주가배수기반 주식가치평가는 비교기업군의 주가배수를 구한 다음 이 주가배수가 옳은 값이고 분석 대상기업의 주가배수도 이와 동일할 것이라 가정하여 분석 대상기업의 주가를 추정하는 방법이다. 분석 순서는 다음과 같다.

1. 대상기업과 비교 가능한 비교기업군을 선정한다.
1. 비교기업군의 내의 각각의 주식의 주가배수를 구한다.
1. 비교기업군의 주가배수 평균을 구한다.
1. 분석 대상기업의 주가배수가 비교기업군의 주가배수 평균과 같다고 가정한다.
1. 주가배수를 재무수치와 곱하여 주가를 산정한다. 

예를 들어 미국의 컴퓨터 생산 기업인 Dell의 주가를 주가배수기반 주식가치평가 방법으로 구해보자. 우선 비교기업으로는 HP와 Lenovo라는 기업을 선정하자. 2008년 기준으로 HP와 Lenovo의 재무제표상 수치와 주가배수는 다음 표와 같다. (재무수치의 단위는 백만달러다.)

| 기업 | 매출 | 순이익 | 장부가 | 시가총액 | PSR | PER | PBR |
| - | -: | -: | -: | -: | -: | -: | -: |
| HP | 84,229 | 7,264 | 38,526 | 115,700 | 1.37 | 15.9 | 3.0 |
| Lenovo | 14,590 | 161 | 1,134 | 6,381 | 0.44 | 39.6 | 5.6 |
| 평균 | | | | | 0.91 | 27.8 | 4.3 |

이 비교기업에 대해 시장이 효율적이어서 Dell의 주가배수도 이 주가배수 수치와 같아야 한다고 가정하면 다음과 같이 Dell의 시가총액을 계산할 수 있다.

| 주가배수 종류 | Dell의 목표 주가배수 값 | Dell의 재무수치 | Dell의 목표 시가총액 |
| - | -: | -: | -: |
| PSR | 0.91 | 매출액: 61,133 |  0.91 x 61,133 = 55,631 |
| PER | 27.8 | 순이익: 2,947 | 27.8 x 2,947 = 81,927 |
| PBR | 4.3 | 장부가: 3,735 | 4.3 x 3,735 = 16,061 |
| 평균 시가총액 | | | (55,631 + 81,927 + 16,061) / 3 = 51,206 |

Dell의 목표 시가총액 51,206(백만달러)을 주식수 2,060(백만주)로 나누면 주가는 51,206 / 2,060 = 24.86 달러가 된다. 참고로 2008년 기준 Dell의 실제주가는 약 25달러였다.

주가배수기반 주식가치평가는 회계지식 등 전문지식이 필요하지 않고 방법이 간단하므로 누구나 쉽게 주가를 계산할 수 있다는 장점이 있다. 하지만 다음과 같은 근본적인 단점으로 인해 평가의 신뢰도가 높지 않다.

- 근본 가정의 오류 : 주가배수기반 주식가치평가는 비교기업군에 대해서만 시장이 효율적이어서 비교기업군의 주가배수는 옳은 값이고 분석 대상기업에 대해서는 시장이 비효율적이므로 대상기업의 현재 주가배수는 옳은 값이 아니라고 가정한다. 하지만 이렇게 구분할 수 있는 논리적인 근거가 필요하다.
- 순환 오류 : 비교기업 B를 사용하여 기업 A의 주가배수기반 주식가치평가를 한 후 반대로 A를 비교기업으로 사용하여 B를 분석한다면 이번에는 B에 대해 시장이 비효율적이고 A에 대해서는 시장이 효율적이라는 반대의 가정을 해야 한다. 
- 비교기업 선정의 어려움 : 실제 현실에서는 어떠한 두 기업도 동일한 조건에서 동일한 사업을 하지 않으므로 올바른 비교기업을 찾는 것이 쉽지 않다. 예를 들어 HP는 Dell과 달리 프린터 사업도 하고 있으며 Lenovo는 다른 두 기업과 달리 중국기업이므로 기업환경이 다르다.
- 상대적 가치평가 : 대상기업의 근본적 가치를 계산하지 않고 오로지 타 기업과의 비교만을 고려한다.
- 주가배수 선정의 어려움 : 사용하는 배수에 따라 추정가격이 달라진다. 위의 Dell 예제에서는 PSR, PER, PBR을 모두 구한 후 이들의 평균값을 사용했지만 각 주가배수에 따른 추정치 차이가 크다.
- 사용불가의 경우 : 재무제표상의 숫자가 음수인 경우 사용이 불가능하다. 예를 들어 적자인 기업은 PER을 쓸 수 없다.

또한 주가배수기반 주식가치평가 방법은 신규상장(IPO) 주식이나 특정한 테마의 주식들이 고평가되는 시점에 서로간 비교를 통해 연쇄적으로 가격 거품을 형성하는 배경이 되기도 하므로 사용상 주의가 필요하다.

### 주가배수기반 주식선별

주가배수기반 주식선별은 주가배수기반 주식가치평가와 달리 분석대상기업과 비교기업을 구분하지 않고 전체 기업군의 모든 주가배수를 상대적으로 비교하여 매수할 주식과 매도할 주식을 선정하는 방법이다. 따라서 주가배수기반 주식가치평가에서처럼 비교기업의 주가배수는 옳고 분석대상 기업의 주가배수는 틀리다고 가정하지 않으며 특정 기업의 주가를 추정하지도 않는다.

주가배수기반 주식선별 방법에서는 각각을 상대적으로 주가배수가 높은 주식을 성장주(growth stock)라고 부르고 상대적으로 주가배수가 낮은 주식을 가치주(value stock)라고 부르기도 한다.

- 성장주
  - 주가배수가 다른 주식보다 상대적으로 높은 주식
  - 재무상 수치에 비해 현재 거래되는 가격이 상대적으로 높음
  - 향후의 성장성을 감안하여 높은 주가배수를 인정
  - 가격 거품이 있을 가능성이 있음
- 가치주
  - 주가배수가 다른 주식보다 상대적으로 낮은 주식
  - 재무상 수치에 비해 현재 거래되는 가격이 상대적으로 낮음
  - 향후의 성장성을 감안하여 높은 주가배수를 인정
  - 향후 재평가를 통해 가격이 상승할 가능성이 있음
 

주가배수기반 주식선별시 일반적으로 사용하는 방법은 주가배수가 높은 주식을 매도(혹은 공매도)하고 주가배수가 낮은 주식을 매수하는 방법이다. 장기투자의 경우 이러한 방법이 성과가 좋을 가능성이 높다고 알려져 있다. 예를 들어 미국주식을 PER, PBR 기준으로 나누어 성과를 분석한 결과를 보자.

다음 표는 전체 미국주식을 1963년부터 2006년까지 매년 초 PER 기준으로 5개의 그룹으로 나눈 뒤 해당 그룹의 1년 평균수익률을 계산한 것이다. 

| PER 그룹 | 그룹내 평균 PER | 해당 그룹의 평균 연수익률 |
|-|-:|-:|
| 5(PER 하위) | 7.1 | 23.2% |
| 4 | 10.8 | 18.1% |
| 3 | 14.7 | 14.9% |
| 2 | 31.3 | 12.1% |
| 1(PER 상위) | 음수 | 13.5% |

PER가 가장 낮은 그룹의 평균 연수익률이 PER가 가장 높은 그룹의 평균 연수익률보다 훨신 높은 것을 볼 수 있다. 다음은 PBR 기준으로 같은 분석을 실시한 결과이다.

| PBR 그룹 | 그룹내 평균 PBR | 해당 그룹의 평균 연수익률 |
|-|-:|-:|
| 5(PBR 하위) | 0.61 | 24.3% |
| 4 | 1.08 | 18.4% |
| 3 | 1.47 | 15.4% |
| 2 | 2.17 | 12.6% |
| 1(PBR 하위) | 4.55 | 9.3% |

마찬가지로 PBR이 가장 낮은 그룹의 평균 연수익률이 PBR이 가장 높은 그룹보다 훨신 높은 것을 볼 수 있다.

이러한 주가배수기반 주식선별법이 항상 좋은 성과를 보장하는 것은 아니다. 주가배수가 낮은 기업에게는 재무제표상에 드러나지 않은 여러가지 악재가 숨겨져 있고 이 정보가 시장에 반영되어 주가가 낮아진 것일 수도 있다. 따라서 주가배수가 낮은 기업에 투자하는 것은 숨겨진 위험을 감수하는 일이고 역사적으로 증명된 높은 수익률은 단지 이러한 위험에 대한 보상일 뿐일 수도 있다. 주식의 위험에 따른 수익률의 관계는 향후 자산가격이론(Asset Pricing Theory)에서 자세히 다루게 된다.

## 자산기반 가치평가 방법론

자산기반 가치평가 방법론은 기업이 소유한 모든 자산(asset)의 시장가치(market value)를 개별적으로 추정하여 합산한 값을 기업의 가치라고 가정하는 방법이다. 재무제표 중 재무상태표(statement of financial position)에는 기업이 소유한 자산과 부채의 항목과 각 항목의 회계적 평가금액이 기록되어 있다. 재무상태표의 자산금액을 모두 합친 후 부채금액을 뺀 값을 순자산가치(net asset value) 또는 장부가치(book-value)라고 한다. 만약 재무상태표에 누락된 항목이 없고 모든 항목이 시장가치로 기록되어 있다면 기업의 장부가치와 시가총액은 일치해야 할 것이다. 즉 앞에서 이야기한 주가배수 중 PBR의 값이 1이 되어야 한다. 

하지만 현실에서는 특정한 자산항목이 재무상태표에 기록되지 않거나 현재 시장가치와 다른 값으로 기록되는 경우가 많다. 이는 회계원칙에 의한 것으로 신뢰성의 원칙(reliability principle)에 따르면 "회계처리와 보고는 신뢰할 수 있도록 객관적인 자료와 증거에 의하여 공정하게 처리"하여야 하기 때문이다. 따라서 시장가치를 평가하기 어려운 자산은 역사적 원가로 기록하거나 일반적이지 않은 무형자산은 누락할 수도 있다. 예를 들어 브랜드 가치, 경쟁기업이 가지지 못한 전문적인 생산기술의 가치, 기업의 종업원들이 가진 전문적 지식의 가치, 기업이 소유한 마케팅 네트워크의 가치, 유통 채널 등의 가치 등은 평가가 어렵고 평가 주체에 따라 크게 달라질 수 있다. 만약 이러한 항목의 시장가치를 올바르게 평가할 수 있다면 전체 기업의 가치를 다음과 같이 구할 수 있다. 

1. 대차대조표에 나열된 자산 및 부채 항목 각각의 현재 시장가치 추정
1. 누락된 자산 및 부채 항목을 식별하고 이들 각각의 시장가치 추정
1. 자산 및 부채 항목을 모두 고려하여 기업의 전체 시장가치 계산

이 방법은 주가배수방법처럼 타 기업과 비교하지 않고 해당 기업의 정보만을 사용하므로 기업의 절대적 가치를 평가할 수 있다는 장점이 있다. 하지만 다음과 같은 이유로 기업의 내부자나 해당 분야의 전문가가 아니면 실제로 수행하기가 아주 어렵다.

- 기업이 소유한 자산의 세부적인 항목은 공시정보나 재무제표에 구체적으로 드러나지 않은 기업의 내부정보인 경우가 많기 때문에 내부인이 아니면 입수하기가 어렵다.
- 거래시장이나 유동성이 없는 자산이나 무형자산의 경우 해당 분야의 가치평가 전문가가 아니면 시장가치를 구하기 어렵고 평가자에 따라 크게 달라질 수 있다.
- 자산의 단순한 판매가격을 사용하는 경우 이러한 판매가격은 해당 회사에서의 실제 유용가치와 일치하지 않을 수 있다. 
- 개별적 자산의 가치가 아닌 그 자산들이 해당 회사에서 같이 사용될 때의 시너지 가치를 추정하기 어렵다.

따라서 자산기반 가치평가 방법론은 일반인들이나 기업 외부의 애널리스트가 사용하기에는 너무 어렵고 평가자마다 크게 다른 값이 나올 수 있어 평가의 객관성을 확보하기도 힘들기 때문에 널리 사용되지 않고 있다.

## 펀더멘탈 분석 방법론

펀더멘탈 분석은 기업이 생성할 미래의 가치를 예측하고 이를 적정한 요구수익률(required rate of return)로 할인한 값을 기업의 가치라고 가정하는 방법이다. 이 방법은 고전적인 현금흐름할인법(cashflow discount method)을 기업가치 평가에 맞게 수정한 것이다. 따라서 펀더멘탈 분석의 원리를 이해하기 위해서는 일단 현금흐름할인법을 알아야 한다.

### 현금흐름할인법 (Cashflow Discount Method)

현금흐름할인법은 투자 후 미래에 받게 될 현금을 현재가치로 할인하여 투자의 가치를 구하는 방법이다. 투자되는 돈(나가는 돈)이나 투자 후 받는 돈(들어오는 돈)의 집합을 <b>현금흐름(cashflow)</b>라고 한다. 보통 나가는 돈은 음수로, 들어오는 돈은 양수로 표시한다.

예를 들어 100만원을 투자한 후 1년마다 1만원을 받고 5년 후에는 101만원을 받으면서 종료되는 투자 A를 생각해보자. 투자 A의 현금흐름은 다음과 같다.

| 시점(년) | 현금흐름 |
|-|-:|
| 0 | -1,000,000 |
| 1 | 10,000 |
| 2 | 10,000 |
| 3 | 10,000 |
| 4 | 10,000 |
| 5 | 1,010,000 |

그런데 1년 후에 받게 될 만원의 가치는 현재의 만원의 가치와 같지 않다. 지금 만원을 은행에 넣어둔다고 하면 1년 후에는 만원에 은행이 주는 이자가 더해져서 더 큰 금액이 될 것이므로 1년 후의 만원은 현재의 만원 보다 가치가 적다. 

$$
\text{현재의 만원의 가치} > \text{1년 후의 만원의 가치}
$$

만약 은행의 이자율이 1%이라고 가정하면 1년 후에 은행으로부터 만원을 받기 위해서 지금 은행에 넣어야 하는 돈은 다음과 같이 계산할 수 있다.  (원 이하 반올림 사용)

$$
\frac{10,000}{1 + 0.01} = 9,901 
$$

즉, 지금 9,901원을 은행에 넣으면 1년 후에는 만원을 받게 된다.

$$
9,901 \times (1 + 0.01) = 10,000
$$

2년 후의 만원의 현재 가치는 연복리에 따라 다음과 같이 계산한다. 이를 <b>할인(discount)</b>한다고 한다.

$$
\frac{10,000}{(1 + 0.01)^2} = 9,803
$$

현금흐름할인법은 이러한 방식으로 미래 현금흐름을 할인하여 투자의 현재가치를 구하는 방법이다. 투자 A의 경우 현금흐름할인법에 따라 현재가치는 투자금액과 일치하는 100만원이 된다.

$$
\frac{10,000}{1 + 0.01} +\frac{10,000}{(1 + 0.01)^2} + \frac{10,000}{(1 + 0.01)^3} + \frac{10,000}{(1 + 0.01)^4} + \frac{1,010,000}{(1 + 0.01)^5} = 1,000,000
$$

따라서 투자 A는 위에서 제시한 현금흐름이 100% 확실하게 보장된다고 하더라도 은행에 저축하는 것에 비해 나은 점이 없다고 할 수 있다.

이번에는 다음과 같은 현금흐름을 가지는 투자 B에 대해 투자의사결정을 해보자.

| 시점(년) | 현금흐름 |
|-|-:|
| 0 | -1,000,000 |
| 1 | 5,000 |
| 2 | 15,000 |
| 3 | 5,000 |
| 4 | 15,000 |
| 5 | 1,000,000 |

투자 B를 집행할지 안할지에 대한 의사결정을 하는 방법은 현금흐름할인법에 따라 은행 이자율로 미래의 현금흐름을 할인하여 현재가치를 계산하여 이 현재가치가 투자금액보다 크면 투자를 하고 현재가치가 투자금액보다 작으면 투자를 하지 않는 것이다. 

은행 이자율 1%로 투자 B의 현재가치를 구하면 990,388원이다.

$$
\frac{5,000}{1 + 0.01} +\frac{15,000}{(1 + 0.01)^2} + \frac{5,000}{(1 + 0.01)^3} + \frac{15,000}{(1 + 0.01)^4} + \frac{1,000,000}{(1 + 0.01)^5} = 990,388
$$

이는 은행을 이용할 경우 다음과 같이 100만원보다 적은 돈을 넣어서 투자 B와 동일한 미래 현금흐름을 만들어 낼 수 있음을 의미한다. 

* $4,950 \times (1 + 0.01) = 5,000$
* $14,704 \times (1 + 0.01)^2 = 15,000$
* $4,853 \times (1 + 0.01)^3 = 5,000$
* $14,415 \times (1 + 0.01)^4 = 15,000$
* $951,466 \times (1 + 0.01)^5 = 1,000,000$
* $ \text{합계} = 4,950 + 14,704 + 4,853 + 14,415 + 951,466 = 990,388 $

### 내부수익률(IRR: Internal Rate of Return)

 다음과 같은 현금흐름을 가지는 투자 C를 살펴보자.

| 시점(년) | 현금흐름 |
|-|-:|
| 0 | -1,000,000 |
| 1 | 10,000 |
| 2 | 20,000 |
| 3 | 30,000 |
| 4 | 40,000 |
| 5 | 1,050,000 |


투자 C의 미래 현금흐름을 이자율로 할인하여 현재가치를 구하면 다음과 같이 투자금액 100만원보다 큰 금액이 나온다.

$$
\frac{10,000}{1 + 0.01} +\frac{20,000}{(1 + 0.01)^2} + \frac{30,000}{(1 + 0.01)^3} + \frac{40,000}{(1 + 0.01)^4} + \frac{1,050,000}{(1 + 0.01)^5} = 1,096,103
$$

따라서 투자 C는 은행 이자율 1%보다 더 수익률이 높다고 할 수 있다. 그러면 투자 C의 수익률 $x$를 숫자로 구하면 얼마일까? 은행 이자율이 아닌 수익률 $x$로 할인하였을 때 현재가치가 투자금액 100만원과 같아지는 할인 수익률 $x$를 찾아야 한다. 따라서 다음 방정식을 풀어 $x$를 구한다. 이렇게 구한 수익률을 투자의 <b>내부수익률(IRR: Internal Rate of Return)</b>이라고 한다. 

$$
\frac{10,000}{1 + x} +\frac{20,000}{(1 + x)^2} + \frac{30,000}{(1 + x)^3} + \frac{40,000}{(1 + x)^4} + \frac{1,050,000}{(1 + x)^5} = 1,000,000
$$

In [1]:
import numpy_financial as npf

npf.irr([-100, 1, 2, 3, 4, 105])

0.02942029471157004

numpy_financial 패키지의 `irr` 함수를 이용하면 투자 C의 내부수익률은 약 2.9%로 은행 이자율 1%보다 두 배 이상 높다는 것을 알 수 있다. 

### 요구수익률 (RRR: Required Rate of Return)

다시 투자 A를 살펴보자. 투자 A의 현재가치를 투자금 100만원과 동일하고 내부수익률은 은행 이자율과 같은 1%이다. 그런데 만약 투자 A에서 약속한 현금흐름이 나오지 않을 위험이 있다면 은행 이자율과 동일한 수익류로는 아무도 투자하지 않을 것이다. 하지만 투자 C처럼 은행 이자율보다 수익률이 훨씬 높다면 위험을 부담하고서라도 투자하고자 하는 사람이 있을 수 있다. 

어떤 위험한 투자가 있을 때 기준이 되는 수익률 값이 정해져 있어 이 기준값보다 투자의 내부수익률이 높으면 투자를 하고 이 기준값보다 투자의 내부수익률이 낮으면 투자를 하지 않는 기준값을 해당 투자의 <b>요구수익률(RRR: Required Rate of Return)</b>이라고 한다.

위험 투자의 요구수익률을 정하는 합리적인 방법은 다른 투자자들이 가진 요구수익률을 모방하는 것이다. 만약 자신의 가진 요구수익률이 다른 투자자들이 가진 요구수익률보다 너무 높으면 다른 투자자들이 하는 투자 기회를 놓칠 것이고 자신의 요구수익률이 다른 투자자들의 요구수익률보다 너무 낮으면 손실을 볼 수 있다. 특히 주식 투자의 경우에는 해당 주식에 투자하는 사람이 많아질수록 투자수익률이 높아지기 때문에 다른 투자자들의 의사결정 기준을 아는 것이 매우 중요하다.


### 펀더멘탈 분석의 개념

주식의 펀더멘탈 분석도 앞서 설명한 현금흐름할인법과 같은 원리를 사용하여 주식의 현재가치 혹은 수익률을 구하고 매수/매도에 대한 의사판단을 한다. 다만 미래의 현금흐름 대신 기업이 미래에 창출할 것으로 예상되는 가치(value)를 할인하여 사용하게 된다. 이러한 미래 가치를 구하기 위해서는 우선 여러가지 정보를 사용하여 <b>미래재무제표(Pro Forma Statement)</b>를 예측해야 한다. 이렇게 예측한 미래재무제표의 값을 사용하여 미래에 창출할 가치를 추정할 수 있다. 가치를 추정하는데 어떤 재무제표 항목을 사용하는 가는 펀더멘탈 분석 모형의 종류에 따라 달라진다. 널리 알려진 펀더멘탈 분석 모형들로는 다음과 같은 것들이 있다.

- 배당할인모형 (Dividend Discount Model)
- 현금흐름할인모형(Discounted Cash-Flow Model) 
- 초과이익모형(Residual Income Model) 
- 초과이익성장모형(Abnormal Earnings Growth Model)

다음 장부터는 이러한 모형에 대해 보다 상세히 설명한다. 

이렇게 구한 가치를 해당 주식에 대한 요구수익률로 할인하여 현재가치를 구한다.

$$
\text{주식의 현재가치} 
= \dfrac{\text{1년후 생성될 가치}}{\text{1 + 요구수익률}}
+ \dfrac{\text{2년후 생성될 가치}}{\text{(1 + 요구수익률)}^2}
+ \dfrac{\text{3년후 생성될 가치}}{\text{(1 + 요구수익률)}^3}
+ \dfrac{\text{4년후 생성될 가치}}{\text{(1 + 요구수익률)}^4}
+ \cdots
$$

특정 주식의 요구수익률은 <b>자산가격이론(asset price theory)</b>를 사용하여 구한다. 가장 기본적인 자산가격이론은 <b>CAPM(Capital Asset Pricing Model)모형</b>으로 개별주식과 시장전체의 수익률간 관계를 나타내는 <b>베타(beta)</b>라는 값을 사용한다. 자산가격이론에 대한 장에서 이에 대해 보다 자세한 내용을 공부할 것이다. 


이렇게 구한 현재가치가 현재의 주가보다 높다면 매수한다. 주식의 경우 주식의 현재가치가 현재 주가와 같아지게 만드는 수익률을 <b>기대수익률(expected return)</b>이라고 한다. 수익률 기준으로 기대수익률이 요구수익률보다 높으면 매수한다. 기대수익률과 요구수익률의 차이를 <b>알파(alpha)</b>라고 부른다. 

펀더멘탈 분석의 단계를 다시 정리하면 다음과 같다.

1. 기업에 대한 다양한 정보로부터 해당 기업의 미래재무제표를 예측한다.
1. 미래재무제표에서 기업이 창출할 미래의 가치를 계산한다.
1. 해당 기업에 대한 요구수익률을 구한다.
1. 요구수익로 미래의 가치를 할인하여 기업의 현재가치를 구한다.
1. 현재가치와 현재 주가를 비교하여 매수/매도를 판단한다.